src/tests/internal/test_azure.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2024 Recursive AI
+
+import os
+from unittest.mock import AsyncMock, Mock
+
+import pytest
+
+from recursiveai.benchmark._internal._llm._llm_model import ChatMessage
+from recursiveai.benchmark._internal._llm._azure_openai_gpt_model import (
+    AZURE_GPT,
+    AzureGPTX,
+)
+
+
+@pytest.fixture(autouse=True, scope="module")
+def skip_if_api_key_not_defined():
+    if os.getenv("OPENAI_API_KEY") is None:
+        pytest.skip("Skipping Azure OpenAI test since OPENAI_API_KEY is not defined")
```

Comment:
It should be AZURE_OPENAI_API_KEY if I'm not mistaken?

Reply:
Yes, the code I added was wrong. I changed to  `AZURE_OPENAI_API_KEY`